Formula/vault.rb

```diff
@@ -40,12 +29,6 @@ def install
 
     ENV.prepend_create_path "PATH", buildpath/"bin"
```

Comment:
Is this still needed?

Reply:
Build errors if it is removed.
```
-->    darwin/amd64: github.com/hashicorp/vault
cp: directory /private/tmp/vault-20171222-45675-j9yhoh/bin does not exist
make: *** [dev] Error 1
```

Reply:
try just `(buildpath/"bin").mkpath` and then maybe this gets reported lol

Reply:
> `(buildpath/"bin").mkpath`

Works.

Reply:
#sad